diff -uraN libxml2-2.15.1/python/setup.py.in omv-libxml2-2.15.1/python/setup.py.in
--- libxml2-2.15.1/python/setup.py.in	2025-10-16 17:49:18.000000000 +0200
+++ omv-libxml2-2.15.1/python/setup.py.in	2025-12-16 00:28:10.594168182 +0100
@@ -92,7 +92,7 @@
 
 # those are added in the linker search path for libraries
 libdirs = [
-os.path.join(ROOT,'lib'),
+# os.path.join(ROOT,'lib'), -L/usr/lib is EVIL
 ]
 
 xml_files = ["libxml.c", "libxml.py", "libxml_wrap.h", "types.c",
